client/extensions/auto-translator/src/widget/translation-dialog/compare-accordion/index.tsx

```diff
@@ -199,11 +181,12 @@ export const CompareAccordion = () => {
             </GridList>
           </div>
           {compareLeft && compareRight && (
-            <div
+            <article
               style={{ width: "100%" }}
-              role="document"
               tabIndex={0}
-              aria-label="Compare Writethrus"
+              aria-label={`${capitalize(gettext("compare"))} ${capitalize(
+                gettext("writethru")
+              )} ${capitalize(gettext("diff"))}`}
```

Comment:
Don't you think spelling out the whole word as "difference" would be better to understand?

Reply:
Client was fine with this during demo. Once they get a chance to use the widget and make some notes, they'll provide feedback on things they want changed.